resdk/query.py

```diff
@@ -309,17 +311,31 @@ def filter(self, **filters):
         new_query._add_filter(filters)  # pylint: disable=protected-access
         return new_query
 
-    def delete(self, force=False):
-        """Delete objects in current query."""
+    def delete(self, force=False, delete_content=False):
+        """Delete objects in current query.
+
+        :param bool force: Do not trigger confirmation prompt. WARNING: Be
+            sure that you really know what you are doing as deleted objects
+            are not recoverable.
+        :param bool delete_content: Also delete all the objects that the
+            current object contains.
+
+        """
+        kwargs = {'force': True}
+        if delete_content and isinstance(self.resource, resdk.resources.collection.BaseCollection):
```

Comment:
Raise an error if the object is not a collection.

Reply:
Done.